app/src/main/res/layout-land/login_activity.xml

```diff
@@ -0,0 +1,77 @@
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context="com.quarkworks.apartmentgroceries.auth.LoginActivity">
+
+    <!-- toolbar -->
+    <include layout="@layout/main_toolbar" />
+
+    <RelativeLayout
+        android:layout_width="@dimen/login_sign_up_username_password_width"
+        android:layout_height="wrap_content"
+        android:layout_below="@+id/main_toolbar_id"
+        android:layout_centerInParent="true">
+
+        <!-- username -->
+        <EditText
+            android:id="@+id/login_activity_username_id"
+            android:layout_width="match_parent"
+            android:layout_height="@dimen/login_sign_up_username_password_height"
+            android:layout_marginTop="@dimen/login_margin_top_land"
+            android:layout_marginLeft="@dimen/login_sign_up_username_password_margin"
```

Comment:
You don't have to use dimens file for everything. We typically use it for very generic things (small, medium, large spacing; small, medium, large text).